doc/warm-reboot/system-warmboot.md

```diff
@@ -0,0 +1,46 @@
+# System-wide Warmboot
+
+# going down path
+
+- stop bgp docker 
+  - enable bgp graceful restart
+  - same as fast-reboot
+- stop teamd docker
+  - just kill
+  - same as fast-reboot
```

Comment:
Is it exactly the same as fast reboot, or some extra state processing is needed for avoid process being killed in the middle of something?